src/main/java/seedu/address/logic/parser/OpenCommandParser.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG_NAME;
+
+import java.util.stream.Stream;
+
+import seedu.address.logic.commands.OpenCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.tag.TagName;
+
+public class OpenCommandParser implements Parser<OpenCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the OpenCommand
+     * and returns an OpenCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
```

Comment:
is the word 'conform' a typo or it is the intended meaning?

Reply:
conform
/kənˈfɔːm/
verb
comply with rules, standards, or laws.